runtime/doc/autocmd.txt

```diff
@@ -466,6 +466,16 @@ CompleteDone			After Insert mode completion is done.  Either
 				CompleteDonePre if you need it.
 				|v:completed_item| gives the completed item.
 
+				Sets these |v:event| keys:
+				    reason	Reason for completion being
+						done. Can be one of:
+						- "accepted": completion was
+						  accepted using |complete_CTRL-Y|.
+						- "aborted": completion was aborted
```

Comment:
- "cancel" is closer to the semantics of CTRL-E.
    - "cancel" means user initiated dismissal.
    - "abort" means "cancel with failure signal".
- using present tense the usual questions about conjugations. that is a general guideline.
- in other places (`:help vim.ui.input()`) we use "confirm", but those should probably be updated to "accept".

```suggestion
						- "accept": completion was
						  accepted using |complete_CTRL-Y|.
						- "abort": completion was aborted
```

Reply:
If we go with `accept`, we can use `cancel` without worrying about AE/BE. Let's do that then.